UI/MapUI.gd

```diff
@@ -60,7 +61,7 @@ func _add_torch_to_current_location():
 	current_player_room.room.set_torch_active()
 
 # Test function
-func _add_light_to_rooms(floor_index: int, room_index: int) -> void:
+func _test_add_light_to_rooms(floor_index: int, room_index: int) -> void:
```

Comment:
if it's a test function, why is not in a test file ? or is more of a WIP function ?

Reply:
Yeah, just wanted to see what the light would look like having torches in several places. This'll move to a test file eventually.